style/pages/projects-details.css

```diff
@@ -24,6 +27,127 @@
   height: calc(100vh - 130px);
 }
 
+.projects-details .container {
+  --container-padding: 1.8rem;
+  gap: 1rem;
+  overflow-x: hidden;
+}
+
+.projects-details .project > *:not(:last-child) {
+  margin-bottom: 2.5rem;
+}
+
+.projects-details .project .content-title {
+  margin-bottom: 0.8rem;
+}
+
+.projects-details .project .header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.projects-details .project .header .mini-profile .title {
+  font-size: 1.25rem;
+}
+
+.projects-details .project .details {
+  overflow-x: auto;
+}
+
+.projects-details .project .details .content {
+  display: flex;
+  align-items: center;
+  gap: 5rem;
+}
+
+.projects-details .project .description .task-description {
+  margin-top: -5px;
+  margin-bottom: -5px;
+}
+
+.projects-details .project .checklist .progress-bar {
+  margin-bottom: 1rem;
+}
+
+.projects-details .project .checklist .tasks {
+  display: flex;
+  flex-direction: column;
+  gap: 3px;
+}
+
+.projects-details .project .checklist .tasks .add-task {
+  margin-left: auto;
+}
+
+.projects-details .activity .tabs {
+  margin-left: -1.8rem;
+  margin-right: -1.8rem;
+  padding: 0 1.8rem;
```

Comment:
Why **these** numbers exactly?!
<details>It's a rhetorical question, don't answer. Just fix it.</details>